templates/store-lists.php

```diff
@@ -1,7 +1,7 @@
 <?php
 global $post;
 
-$pagination_base = str_replace( $post->ID, '%#%', esc_url( get_pagenum_link( $post->ID ) ) );
+$pagination_base = empty($post) ? '': str_replace( $post->ID, '%#%', esc_url( get_pagenum_link( $post->ID ) ) );
```

Comment:
kindly fix the formatting issue: eg: `empty( $post ) ? '' : `

Reply:
Thanks mate. I applied the fix.